src/components/literatureReview/LiteratureReview.js

```diff
@@ -1,26 +1,44 @@
 import React from 'react';
```

Comment:
Likewise here, no direct changes by me. The only changes made were running `npm run lint`.